torchao/prototype/models/gpt_fused/README.md

```diff
@@ -0,0 +1,13 @@
+## gpt-fused
+
+A more handwritten version of [gpt-fast](https://github.com/pytorch-labs/gpt-fast)'s model.py for us to experiment with.
```

Comment:
wdym by more handwritten?

Reply:
We could use this to try various fused kernels (Triton or CUDA).